Add zoolz

Nicos18 opened the pull request.

Nicos18 committed:
Update backup.yml

mxxcon commented on _data/backup.yml:
```diff
@@ -180,7 +188,8 @@ websites:
 
-  - name: Sync
+
+- name: Sync
```
Please undo this change. This breaks YAML format

mxxcon reviewed (changes requested):
Please add only 1 site per pull request. Either Degoo or Zoolz.
Whichever site you'll decide to keep in this pull request, please also add 32x32 logo file.

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Add files via upload

stephengroat reviewed (dismissed):
thanks for the pr @Nicos18

mxxcon reviewed (changes requested):
What about http://wiki.zoolz.com/how-do-i-enable-two-factor-authentication-on-my-zoolz-intelligent-account/ ?

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Update backup.yml

Nicos18 committed:
Update backup.yml

stephengroat committed:
whitespace cleanup

Jawshy committed:
Backup: Zoolz: Lower case for filename

Jawshy committed:
Backup: Zoolz: Update url

Use HTTPS

Change subdomain to the redirect

Jawshy committed:
Backup: Zoolz: Add software

Zoolz uses Time-based One-Time Password algorithm (TOTP) according to the doc link

mxxcon commented on _data/backup.yml:
```diff
@@ -236,3 +236,12 @@ websites:
     doc: http://www.zetta.net/about/blog/how-set-zetta-two-factor-authentication/
+
+  - name: Zoolz
+    url: https://intelli.zoolz.com/
```
I'm not sure we should use intelli subdomain. I think it's better to use www or just apex and let them redirect wherever they'll decide to...

stephengroat committed:
Merge branch 'master' into patch-1

mxxcon commented on _data/backup.yml:
```diff
@@ -231,3 +231,12 @@ websites:
+    tfa: Yes
+    software: Yes
+    exceptions:
+        text: "Only available for business accounts."
```
Is it still restricted only to biz accounts?

Nicos18 replied:
@mxxcon Yes, unfortunately
